### PR TITLE
Change stash list styling

### PR DESCRIPTION
**Pull request by MarvinKweyu**
The idea is pretty simple actually. For someone who uses `stash` often, it can get difficult to read the stash listing as it grows in size.
So instead of
```
stash@{2}:On master: Modify readme to reflect config change
```
You would now have:
```
stash@{2}:   82 minutes ago  On master:Modify readme to reflect config change
```
The sections are tabulated for the sake of readability and have different color per tabular column just for clarity.
Take a look at the change and let me know?

### CODE REVIEW

**Comment by diogocavilha** [comment]
Yes, that's really good! I like that. Mainly that time information.

I'd ask you to change the version file as well ([version.sh](https://github.com/diogocavilha/fancy-git/blob/master/version.sh)), and add those changes into the changelog file ([CHANGELOG.md](https://github.com/diogocavilha/fancy-git/blob/master/CHANGELOG.md))

By the way... I've just seen that I forgot to change the version file last time I changed fancygit :(
I've changed the changelog but I didn't change the version file, my bad!

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 
 > Any trouble, please visit: https://github.com/diogocavilha/fancy-git#troubleshooting-pick
 
+
+### v6.2.9
+
+- Add styling to git stash listing  for clarity
+
+### v6.2.8
```
Comment (MarvinKweyu) [inline CHANGELOG.md]: I added the change from the previous update on centralizing config files to the `changelog`. Followed the consistency from the logs in `v5....`.
So this new update would come in as a `v.6.2.9` and so on.

**Comment by diogocavilha** [comment]
Everything is ok now. The last thing I would ask you to do is to create a release tag following this pattern... [releases](https://github.com/diogocavilha/fancy-git/releases)

File: CHANGELOG.md (hunk 1 above)
Comment (diogocavilha) [reply CHANGELOG.md]: That's right.

**Comment by MarvinKweyu** [comment]
Like so? [v6.2.9](https://github.com/MarvinKweyu/fancy-git/releases)

**Comment by diogocavilha** [comment]
@MarvinKweyu 
Thank you for your contribution!

**Pull request merged**